meinberlin/apps/plans/models.py

```diff
@@ -16,9 +16,10 @@
 from adhocracy4.projects import models as project_models
 from adhocracy4.projects.enums import Access
 from adhocracy4.projects.fields import TopicField
+from adhocracy4.projects.models import ProjectContactDetailMixin
 
 
-class Plan(UserGeneratedContentModel):
+class Plan(ProjectContactDetailMixin, UserGeneratedContentModel):
```

Comment:
when using the mixin, the old field needs to be removed from the model (and will be replaced by the new field, I think). Somehow the migration is right and only adds fields, though.